www/checkup/main.js

```diff
@@ -720,12 +725,11 @@ define([
     });
 
     assert(function (cb, msg) { // FIXME possibly superseded by more advanced CSP tests?
-        var url = `/common/onlyoffice/dist/${OOCurrentVersion.currentVersion}/web-apps/apps/spreadsheeteditor/main/index.html`;
-        msg.appendChild(CSP_WARNING(url));
+        msg.appendChild(CSP_WARNING(sheetURL));
```

Comment:
Is no `if (ooEnabled) {` needed here?

Reply:
Fixed! Typo from when i reverted the Prettier formatting.